requirements/extra.txt

```diff
@@ -3,7 +3,7 @@
 # You can also include it in local.txt for local development
 
 # Engines
--r engine/bigquery.txt
+# -r engine/bigquery.txt
```

Comment:
This dependency list causes the docker build to silently fail. Therefore, it has been removed.